src/test/java/com/google/sps/ChallengeTest.java

```diff
@@ -34,11 +40,6 @@ public void isEqualValidDiffInstance(){
     Assert.assertTrue(ChallengeData.challenges[1].isEqual(challenge_list.get(1)));
   }
 
-  @Test
-  public void isEqualInvalid(){
```

Comment:
Why did we drop this test?

Reply:
thanks for the catch. that was not meant to be dropped